frontend/src/hooks/entities/auth-hooks.ts

```diff
@@ -61,7 +62,10 @@ export const useLogout = (
 
       return await apiClient.logout();
     },
-    onSuccess: () => {},
+    onSuccess: async () => {
+      queryClient.removeQueries([CURRENT_USER_KEY]);
+      await logoutRedirection();
+    },
```

Comment:
I have this minor suggestion 
```suggestion
onError: (error) => {
  toast.error(t("message.logout_failed"));
}
    },
```

Reply:
+1

Reply:
@IkbelTalebHssan good idea i'll address it in different PR